From e770891392f6402b50a0905e2bac32597acc21d4 Mon Sep 17 00:00:00 2001
From: Christopher Degawa <ccom@randomderp.com>
Date: Wed, 16 Sep 2020 01:26:21 +0000
Subject: [PATCH] srt: Replace SRTO_TSBPDDELAY with SRTO_LATENCY

SRTO_TSBPDDELAY was completely removed in
https://github.com/Haivision/srt/commit/0e2201aff6b379979cec43fee5e8f162717f6346
and it seems SRTO_LATENCY is it's alias that's still in the code

Signed-off-by: Christopher Degawa <ccom@randomderp.com>
---
 modules/access/srt.c        | 2 +-
 modules/access_output/srt.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/modules/access/srt.c b/modules/access/srt.c
index 499a300196..d7ef51fccf 100644
--- a/modules/access/srt.c
+++ b/modules/access/srt.c
@@ -164,7 +164,7 @@ static bool srt_schedule_reconnect(stream_t *p_stream)
 
     /* Set latency */
     srt_set_socket_option( strm_obj, SRT_PARAM_LATENCY, p_sys->sock,
-            SRTO_TSBPDDELAY, &i_latency, sizeof(i_latency) );
+            SRTO_LATENCY, &i_latency, sizeof(i_latency) );
 
     /* set passphrase */
     if (psz_passphrase != NULL && psz_passphrase[0] != '\0') {
diff --git a/modules/access_output/srt.c b/modules/access_output/srt.c
index 0b66f24a8a..3d579eb1bd 100644
--- a/modules/access_output/srt.c
+++ b/modules/access_output/srt.c
@@ -155,7 +155,7 @@ static bool srt_schedule_reconnect(sout_access_out_t *p_access)
 
     /* Set latency */
     srt_set_socket_option( access_obj, SRT_PARAM_LATENCY, p_sys->sock,
-            SRTO_TSBPDDELAY, &i_latency, sizeof(i_latency) );
+            SRTO_LATENCY, &i_latency, sizeof(i_latency) );
 
     /* set passphrase */
     if (psz_passphrase != NULL && psz_passphrase[0] != '\0') {
-- 
2.25.1

